File: src/main/java/guru/xgm/language/jass/psi/tree/JassIToken.java
package guru.xgm.language.jass.psi.tree;

import com.intellij.psi.tree.IElementType;
import guru.xgm.language.jass.lang.JassLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class JassIToken extends IElementType {

    public JassIToken(@NotNull @NonNls String debugName) {
        super(debugName, JassLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "JassIToken." + super.toString();
    }
}

File: src/main/kotlin/guru/xgm/language/jass/psi/tree/JassIToken.kt
package guru.xgm.language.jass.psi.tree

import com.intellij.psi.tree.IElementType
import guru.xgm.language.jass.lang.JassLanguage.Companion.instance
import org.jetbrains.annotations.NonNls

class JassIToken(debugName: @NonNls String) : IElementType(debugName, instance) {
    override fun toString(): String {
        return "JassIToken." + super.toString()
    }
}
